environment.yml

```diff
@@ -59,6 +59,7 @@ dependencies:
 
     # nbextensions
     - jupyterthemes==0.20.0
+    - markus-jupyter-extension==0.1.*
```

Comment:
Given that this is probably a quick moving extension, i'd recommend pinning to a specific version here and not using `*`. That would allow us to more easily trigger a rebuild when a new version is released.